docs/contributing/documentation_guidelines.md

```diff
@@ -157,17 +169,41 @@ some command
 
 </details>
 
+#### Code blocks that contain triple backticks
+
+You can use three or more backticks to create code blocks. If you need to include triple backticks in a code block, you can use a different number of backticks to wrap the code block. This is useful when you need to demonstrate a Markdown code block, such as in the examples above.
+
+`````md
+````md
+```python
+print("Hello, world!")
+```
+````
+`````
+
+
+
+<details>
+
+<summary>Rendered output</summary>
+
+````md
+```python
+print("Hello, world!")
+```
+````
+
+</details>
+
 ### Links
 
 Links are fundamental in documentation. Use internal links to tie your content to other docs, and external links as needed. Pick relevant text for links, so readers know where they will land.
 
-Don’t rely on [`links over code`](https://www.example.com/), as they are impossible to spot.
```

Comment:
Maybe we should refine the nuance here, I always thought this was a good guideline to follow. 

Maybe something like, 

```
Don't let external links hide [critical context](https://www.example.com/) for the reader, instead provide the core information on the page and use links for added context.
```

Reply:
I do not understand what the original sentence meant. It seems to be talking about inline code as links specifically, and I don't find "they are impossible to spot" to be true. Maybe it was a bit harder to spot when the theme used custom fonts, but now with the system fonts combined with the bright underline and the hover effect I think they're not impossible to spot. Otherwise, with this guideline, how would you link to API references?

Your sentence, on the other hand, is much clearer on what makes it not OK. I like it, thanks. I've added your sentence to the section, and added a few examples of anti-patterns.

Reply:
Thanks, hopefully that's useful.